### ISSUE TITLE
I want to be able to browse the contents in a space without having to navigate back to the 'space' page everytime I open an item

### ISSUE THREAD

**Issue by PriPatel** [lightdash/lightdash#3907]
Here's a prototype of this concept:
https://www.figma.com/proto/odMRqKZ30XBli5pTLgS1mx/contentmanagement-pages-draft?node-id=642%3A63142&scaling=min-zoom&page-id=642%3A63141&starting-point-node-id=642%3A63142&hide-ui=1

I think it is annoying to have to go 'back' to the Space page every time you want to view the next item in the list. If there are 3 dashboards in the Space, can't I just view them in one go? This could also help business users curate their Spaces better. You could order the items in the way you would want people to navigate through them. 
I think this might be a solution to https://github.com/lightdash/lightdash/issues/3722 and I don't think it's an alternative to being able to add 'tabs' to dashboards. (this has been requested a few times). 

It's just a concept! Thoughts?

**Comment by TuringLovesDeathMetal** [comment]
This is an interesting idea 👀 

I think the only thing I'm not sure about is like, going down to the 15th dashboard or something in a space, then this feature becomes not nearly as useful (i.e. I only really want to have a "story mode" for a select few dashboards, probably not all of them are ordered in teh folder in a sequence that makes sense). 

Maybe there's like a "story" mode at the top or something? Where you can hit `play` and it shows you the dashboards that have been curated into this story? 

^But this is feeling more like it's solving the same problem as pinning.

**Comment by PriPatel** [comment]
@TuringLovesDeathMetal yes!!! maybe you are just viewing your pinned items in this way. 
Or yeah, you have selected which dashboards are worthy of being part of the story collection. You can view them in full screen without having to hit 'back' every time.

**Comment by TuringLovesDeathMetal** [comment]
Another "similar solution" kinda is our KPI dashboard:

I have the actual KPI dashboard, then I have the 4 other dashboard that "dig into" our metrics. 

In this case, I'd almost prefer those to be in tabs or something (instead of played one after the other).

But something like a `product` or `finance` space might have a bunch of unrelated dashboards you'd want to see one-after the other. 

E.g. if I had a product space, I'd probably want the "product overview" dashboards for a bunch of our key products to show one after the other (so people can get a quick overview of how our key products are doing and dig into them).
